www/docs/build-a-rover.rst

```diff
@@ -0,0 +1,198 @@
+| IR Ear Signal     | C.H.I.P Pin |
++===================+=============+
+| left              | XIO-P2      |
++-------------------+-------------+
+| right             | XIO-P4      |
++-------------------+-------------+
+
+Note: These sensor circuits are not great. Their detection range is only of a couple of inches.
+Our future reference design will include `a PCB with a Silicon Labs I2C MEMS
+IR sensor <https://upverter.com/ductape/aef33f7c39fd29d5/rovercode-prox-sensor/>`_, which should work much better.
+
+webcam -- the eye
+-------------------
+:important note: The default CHIP kernel does not enable the USB Video Class (UVC) driver. In the future we hope to provide a ready-to-use emmc image with this driver included, but for now you'll have to rebuild the kernel with the UVC driver included. This is a more advanced task. Your best bet is `this tutorial <http://www.raspibo.org/wiki/index.php/Compile_the_Linux_kernel_for_Chip:_my_personal_HOWTO>`_. If you're not up for this, don't worry -- just stay tuned for an update to this page telling you where you can get a ready-to-use emmc image.
+
+At the moment, the webcam streaming service is not installed or
+started with the main rovercode service (we have `an issue card
+<https://github.com/aninternetof/rovercode/issues/110>`_ to fix this). So,
+you'll need to get and run mjpg-streamer yourself for now.
+
+Get and build mjpg-streamer by following steps 1 through 5 in `these
+instructions <https://blog.miguelgrinberg.com/post/how-to-build-and-run-mjpg-streamer-on-the-raspberry-pi>`_.
+
+To make mjpg-start on boot, add this line to `/etc/rc.local`. Replace {BUILD_DIR} with the absolute path to the directory where you built
+mjpg-streamer.
+
+.. code-block:: guess
+
+    {BUILD_DIR}/mjpg_streamer -i "{BUILD_DIR}/input_uvc.so" -o "{BUILD_DIR}/output_http.so -w {BUILD_DIR}/www"
+
+Restart the rover. You can check that mjpg-streamer has started by
+pointing your PC's browser at `{ip-address-of-your-rover}:8080`. You should see
+the mjpg-streamer demo page.
+
+assembly
+----------
+Here is how it all hooks together:
+
+.. image:: http://i.imgur.com/h9Y6mPG.png
+
+Put everything on the chassis how you see fit. Below are some
+photos of how we did it. Hot glue is your friend.
+
+.. image:: http://i.imgur.com/p3TpMNj.jpg
+.. image:: http://i.imgur.com/N0N6NQe.jpg
+.. image:: http://i.imgur.com/TsyoME6.jpg
+
+Note that the motors are still powered by the Thunder Tumbler AA
+battery pack, so make sure there are batteries in there and
+that the switch on the bottom is turned on when in use.
+
+The webcam draws too much current to be directly connected to the
+C.H.I.P's USB host port. So, we use a powered USB hub.
+
+install rovercode service
+--------------------------
+Connect to the C.H.I.P. via serial or SSH.
+
+Follow the Standard Setup on the `quickstart page <quickstart.html>`_.
+
+play
+------
+Go to `<rovercode.com>`_, sign up for an account, then go to `Mission Control
```

Comment:
I think you need the `https://` here for the link to work correctly for `rovercode.com`.